wallet-new/src/Cardano/Wallet/API/V1/LegacyHandlers/Transactions.hs

```diff
@@ -37,33 +35,15 @@ handlers submitTx =
              newTransaction submitTx
         :<|> allTransactions
         :<|> estimateFees
+        :<|> newUnsignedTransaction
+        :<|> newSignedTransaction submitTx
```

Comment:
`newUnsignedTransaction` creates a raw transaction (which will be signed externally).
`newSignedTransaction` takes (already) signed transaction and submit it to the blockchain.